Cargo.toml

```
@@ -13,6 +13,7 @@ license = "GPL-3.0+"
 
 [build-dependencies]
 bindgen = "0.72"
+system-deps = "7.0.5"
```

```suggestion
system-deps = "7"
```

Nit: I'd prefer using `system-deps = "7"`. The specific patch release doesn't really matter (and 7.0.5 is already out of date).